src/game/client/water/w3dwater.cpp

```diff
@@ -1483,8 +1482,7 @@ void WaterRenderObjClass::Render_Water_Mesh()
         }
 
         m_vertexBufferD3D->Unlock();
-        Matrix3D m(m_transform);

```

Comment:
I think this needs checking.

Reply:
Looks safe to do. `m_transform` already is `Matrix3D`, so this just creates a useless copy here and `DX8Wrapper::Set_Transform` just reads it.